perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/ExpirationNotifScheduler.java

```diff
@@ -102,6 +106,9 @@ public void setPerun(PerunBl perun) {
 	public ExpirationNotifScheduler() {
 	}
 
+	@Autowired
+	private RegistrarManager registrarManager;
```

Comment:
I would prefer having getter/setter and put ``@Autowired`` above the setter. Also please specify it in ``perun-registrar-lib.xml`` in bean definition.

Reply:
Ok, I changed it like you write above